gondorapi/views/address.py

```diff
@@ -32,3 +33,21 @@ def get_user_addresses(self,request):
         addresses = Address.objects.filter(user= user)
         serializer = AddressSerializer(addresses, many=True)
         return Response(serializer.data)
+    
+
+    @action(detail= True, methods=["delete"], url_path="deleteAddress")
```

Comment:
Could remove deleteAddress url path, so it's just http://localhost:8000/addresses/uuid with the delete method.